defs/bazel_version.bzl

```diff
@@ -0,0 +1,32 @@
+# Copied from https://github.com/tensorflow/tensorflow/blob/a6510237f06f03babfbcce1bcfa77f07f0d03c50/tensorflow/workspace.bzl
+
+# Parse the bazel version string from `native.bazel_version`.
+def _parse_bazel_version(bazel_version):
+  # Remove commit from version.
+  version = bazel_version.split(" ", 1)[0]
+
+  # Split into (release, date) parts and only return the release
+  # as a tuple of integers.
+  parts = version.split("-", 1)
+
+  # Turn "release" into a tuple of strings
+  version_tuple = ()
+  for number in parts[0].split("."):
+    version_tuple += (str(number),)
+  return version_tuple
+
+# Check that a specific bazel version is being used.
+def check_version(bazel_version):
```

Comment:
When is this actually used?

Reply:
the intent is to use it in the k/k `WORKSPACE` - I'll have it verify bazel 0.5.4+ there (which would be a better error message than a syntax error or whatever).